configure-postgis.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+#
+# Enable the PostGIS extension
+#
+
+import os
+import psycopg2
+from psycopg2.sql import Identifier
+import re
+import sys
+from urlparse import urlparse
+
+def identifier(s, connection):
+    """
+    Return s as a double-quoted string (good for psql identifiers)
+    """
+    return Identifier(s).as_string(connection)
+
+def postgis_sql(user, connection):
+    """
+    Return some SQL to configure the PostGIS extension.
+    """
+    
+    template_filename = 'configure-postgis.sql'
+    with open(template_filename) as fp:
+        template = fp.read()
+    return template.format(user=identifier(user, connection))
+
+def get_env(name):
+    if name not in os.environ:
+        raise Exception("Required environment variable %s is missing" % name)
+    value = os.environ[name]
+    if not value:
+        raise Exception("Environment variable %s missing value" % name)
+    return value
+
+def main():
+    database_url = get_env('DATABASE_URL')
+    database = urlparse(database_url)
+    user = database.username
+    
+    try:
+        with psycopg2.connect(database_url) as conn:
+            sql = postgis_sql(user, conn)
```

Comment:
A way to accomplish this without having to add the functions above to template the .sql file: Set a Postgres "customized option"
```
setuser = `SET postgisinstall.user TO ` + user
```
then add 
```
cur.execute(setuser)
```
 just above the `cur.execute(sql)` below. 

Then in the SQL file, change all the references to {user} to reference that customized option. For example
 ```
ALTER SCHEMA tiger OWNER TO {user};
```
changes to 
```
ALTER SCHEMA tiger OWNER TO (SELECT current_setting('postgisinstall.user'))
```